octobot_evaluators/util/evaluation_util.py

```diff
@@ -37,3 +37,42 @@ def get_shortest_time_frame(ideal_time_frame, preferred_available_time_frames, o
         return time_frame_manager.sort_time_frames(preferred_available_time_frames)[0]
     else:
         return time_frame_manager.sort_time_frames(others)[0]
+
+
+def local_trading_context(evaluator, symbol, time_frame, trigger_cache_timestamp,
+                          cryptocurrency=None, exchange=None, exchange_id=None,
+                          trigger_source=None, trigger_value=None):
+    try:
+        import octobot_trading.api as exchange_api
+        import octobot_trading.modes as modes
+        exchange_manager = exchange_api.get_exchange_manager_from_exchange_name_and_id(
+            exchange or evaluator.exchange_name,
+            exchange_id or exchange_api.get_exchange_id_from_matrix_id(evaluator.exchange_name, evaluator.matrix_id)
+        )
+        trading_modes = exchange_api.get_trading_modes(exchange_manager)
+        trading_mode = trading_modes[0]
```

Comment:
I don't think we should manipulate trading mode instances here. It doesn't make sense in the Strategy / Trading Mode current architecture.

Reply:
Right, I just checked, we actually can remove it altogether from context, I'll do it

Reply:
We are now inly using the trading mode class